index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <style>
+    @media (max-width: "700px"){
+  #projects {
+    width: 25%;
+  }
+}
+
+#welcome-section {
+  height: 100vh;
+}
+
+#navbar {
+  position: fixed;
+  top: 0;
+  right: 0;
```

Comment:
You can use external css

Reply:
@Alibaba2023 👋 

- [ ] Kindly phrase the comments politely as suggestions instead of orders.
  - For example:
    - This is a polite suggestion: "I think it would be better to use CSS".
 
- [ ] Kindly use reasoning `why` students need to consider your suggestion. Why do students need to use external CSS? Use reasoning here and you can add any referral link from where students can get hints.

- [ ] Kindly use [Grammarly](https://app.grammarly.com/) so that there is no spelling mistake and no capital letters in the middle of any sentence. It is more professional to check the spelling while giving any suggestion in every aspect of our job and profession.